### PR TITLE
fix: responsiveness on remote provider ui w/signoff

### LINKED ISSUE #10743: Responsiveness on 'Remote Provider' page.
### Current Behavior
 Remote provider capabilities & extensions were recently included in the Meshery UI's preferences, however, there is a responsiveness issue that needs fixing, primarily in mobile view. See attached image.

### Expected Behavior
 A responsive site that accounts for the mobile view of this new page. Here's a possible solution [in Figma]. (https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?type=design&node-id=29694%3A19485&mode=design&t=mElCikdb8NjkLBBo-1)

### Screenshots/Logs
What we have: 
Remote provider issue

What we could have: 
Figma remote provider page

### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by ChenemiKen**
**Notes for Reviewers**

This PR fixes #10743

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — COMMENTED** [review]
Thanks much, @ChenemiKen.

To facilitate review, will you please take a before and after screenshot?

**Comment by Deepak0320Singhal** [comment]
@ChenemiKen Let's discuss this in Meshery Dev meeting on Wednesday. Adding this as an agenda item....
Meeting minutes: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#
Meeting link: https://meet.layer5.io/meshery

**Comment by ChenemiKen** [comment]
> Thanks much, @ChenemiKen.
> 
> To facilitate review, will you please take a before and after screenshot?

**Before:** 

Screenshot 2024-07-03 161450
Screenshot 2024-07-03 161522




**After:**


Screenshot 2024-07-03 161114
Screenshot 2024-07-03 161138
Screenshot 2024-07-03 161157
Screenshot 2024-07-03 161222
Screenshot 2024-07-03 161240
Screenshot 2024-07-03 161255
Screenshot 2024-07-03 161317

**Comment by Deepak0320Singhal** [comment]
@ChenemiKen Thank you for your Cotribution.
Let's discuss this in Meshery Dev meeting. Adding this as an agenda item....
Meeting minutes: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#
Meeting link: https://meet.layer5.io/meshery

File: ui/components/UserPreferences/index.js [hunk 1]
```diff
@@ -457,7 +458,7 @@ const UserPreference = (props) => {
                         <Typography
                           variant="body1"
                           className={props.classes.hideScrollbar}
-                          style={{ marginRight: '20px' }}
+                          style={{ marginRight: '20px', wordBreak: 'break-word' }}
```
Comment (leecalcote) [inline ui/components/UserPreferences/index.js:461]: Thanks, @ChenemiKen!

Generally, we prefer not to break words. Word wrap - ok.

Will you take a screenshot of before and after these changes, @ChenemiKen?

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
@ChenemiKen, are you working to eliminate the guttered whitespace on either side of the tables in mobile view given the feedback on this week's dev meeting?

**Comment by ChenemiKen** [comment]
> @ChenemiKen, are you working to eliminate the guttered whitespace on either side of the tables in mobile view given the feedback on this week's dev meeting?

@leecalcote Yes, I'm working on it.

File: ui/components/UserPreferences/index.js (hunk 1 above)
Comment (ChenemiKen) [reply ui/components/UserPreferences/index.js:461]: noted

**Comment by sudhanshutech** [comment]
@ChenemiKen done?

**Comment by sudhanshutech** [comment]
@ChenemiKen checking again...are you still working on this?

**Pull request closed without merging**